modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -10,36 +10,29 @@
     #liferay.workspace.bundle.cache.dir=~/.liferay/bundles
 
     #
+    # Legacy Feature (See liferay.workspace.product.key)
```

Comment:
Lets not call it Legacy, as we still want to support it going forward.

Instead call it `Override bundle url` or something, so they know this section is for overriding the `liferay.workspace.product`